mne/preprocessing/ica.py

```diff
@@ -1400,6 +1401,63 @@ def plot_components(self, picks=None, ch_type=None, res=64, layout=None,
                                    image_interp=image_interp,
                                    head_pos=head_pos)
 
+    def plot_properties(self, inst, picks=None, axes=None, dB=True,
+                        cmap=None, plot_std=True, topomap_args=None,
+                        image_args=None, psd_args=None, figsize=None,
+                        show=True):
+        """Display component properties: topography, epochs image, ERP,
+        power spectrum and epoch variance.
+
+        Parameters
+        ----------
+        inst: instance of Epochs or Raw
+            The data to use in plotting properties.
+        picks : int | array-like of int | None
+            The components to be displayed. If None, plot will show the first
+            five sources. If more than one components were chosen in the picks,
+            each one will be plotted in a separate figure. Defaults to None.
+        axes: list of matplotlib axes | None
+            List of five matplotlib axes to use in plotting: [topo_axis,
+            image_axis, erp_axis, spectrum_axis, variance_axis]. If None a new
+            figure with relevant axes is created. Defaults to None.
+        dB: bool
+            Whether to plot spectrum in dB. Defaults to True.
+        cmap: matplotlib colormap | None
+            Colormap to use in both topoplot and epochs image. If None topoplot
+            and epochs image use their default "RdBu_r" colormap.
+            Defaults to None.
```

Comment:
I can remove it if you want. However I think it is much more convenient to write `cmap='viridis'` instead of `topomap_args={'cmap': 'viridis'}, image_args={'cmap': 'viridis'}`. After all - all `cmap` does in other functions like `plot_epochs_image` is overriding matplotlib defaults and the user can set colormap easily on their own with `im.set_cmap` for example.